embed_grader/serapis/templates/serapis/testbed_status_list.html

```diff
@@ -0,0 +1,51 @@
+{% extends 'serapis/base_sidebar.html' %}
+{% load staticfiles %}
+
+{% block profile-usertitle-name %}
+  Testbed
+{% endblock %}
+
+{% block profile-usertitle-job %}
+  <br/>
+{% endblock %}
+
+{% block testbed-status-list-active %}active{% endblock %}
+
+{% block main-content %}
+  <div class="main-content">
+    <h1 class="page-header">Testbed Status List</h1>
+
+    {% if not testbed_list %}
+      <p>No testbeds available to view.</p>
+    {% else %}
+      <div class="table-responsive">
+      <table class="table table-hover">
+        <tr>
+          <th>#</th>
+          <th>IP Address & Port</th>
+          <th>Current Status</th>
+          <th>Last Report Time</th>
+          <th>Last Report Status</th>
+          <th>Graded task</th>
+        </tr>
+        {% for testbed in testbed_list %}
+        <tr>
+          <td>{{ forloop.counter }}</td>
+          <td>{{ testbed.ip_address }}</td>
+          <td>{{ testbed.status }}</td>
```

Comment:
For line 35, 37, and 38:

line 35, use `testbed.get_status_display`, otherwise it will print an integer which is less informative. Likewise line 37.
(Reference: https://github.com/nesl/AutoGrader/blob/master/embed_grader/serapis/templates/serapis/homepage.html#L34)

For line 38: Not sure what is being printed here. Can you put course title, assignment name, task name, and submission ID here?